_topic_map.yml

```diff
@@ -279,18 +279,8 @@ Topics:
     File: installing-rhv-user-infra
   - Name: Uninstalling a cluster on RHV
     File: uninstalling-cluster-rhv
-- Name: Installing on oVirt
-  Dir: installing_rhv
-  Distros: openshift-origin
-  Topics:
-  - Name: Installing a cluster quickly on oVirt
-    File: installing-rhv-default
-  - Name: Installing a cluster on oVirt with customizations
-    File: installing-rhv-customizations
-  - Name: Installing a cluster on oVirt with user-provisioned infrastructure
-    File: installing-rhv-user-infra
-  - Name: Uninstalling a cluster on oVirt
-    File: uninstalling-cluster-rhv
```

Comment:
Did you intentionally delete all of the oVirt OKD docs? These were added in #27649 for OKD, and I assume this should remain? Perhaps a merge conflict gone wrong?

Reply:
Yes, I guess I made some mistakes in my merge conflict resolution, I'll try to fix this